tests/suite/foundations/repr.typ

```diff
@@ -38,6 +38,7 @@
 
 // Strings and escaping.
 #raw(repr("hi"), lang: "typc")
+#raw("#" + repr("hi"), lang: "typm")
```

Comment:
I think an example with some actual math would be better.

Reply:
Got it, how is this?